stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/viewholder/internal/MessagePlainTextViewHolder.kt

```diff
@@ -52,6 +56,18 @@ internal class MessagePlainTextViewHolder(
         }
     }
 
+    private fun StreamUiItemMessagePlainTextBinding.configMargins(style: MessageListItemStyle) {
+        messageContainer.updateLayoutParams<ViewGroup.MarginLayoutParams> {
+            marginStart = style.messageStartMargin
+            marginEnd = style.messageEndMargin
+        }
+
+        footnote.updateLayoutParams<ViewGroup.MarginLayoutParams> {
+            marginStart = style.messageStartMargin
+            marginEnd = style.messageEndMargin
+        }
```

Comment:
Should this be performed in a decorator perhaps?

Reply:
I'll create a new one then. "MessageCo trainerMargin"